internal/cluster/transition_test.go

```diff
@@ -870,7 +870,7 @@ var _ = Describe("Refresh Cluster - Advanced networking validations", func() {
 					SufficientMastersCount:              {status: ValidationSuccess, messagePattern: "Cluster has sufficient number of master candidates"},
 					isClusterCidrDefined:                {status: ValidationFailure, messagePattern: "Cluster Network CIDR is undefined"},
 					isServiceCidrDefined:                {status: ValidationFailure, messagePattern: "Service Network CIDR is undefined"},
-					noCidrOverlapping:                   {status: ValidationPending, messagePattern: "At least one of the CIDRS .MachineNetworkCIDR, ClusterNetworkCIDR, ServiceNetworkCKIDR. is undefined"},
+					noCidrOverlapping:                   {status: ValidationPending, messagePattern: "At least one of the CIDRS .MachineNetworkCIDR, ClusterNetworkCIDR, ServiceNetworkCIDR. is undefined"},
```

Comment:
```suggestion
					noCidrOverlapping:                   {status: ValidationPending, messagePattern: "At least one of the CIDRs (Machine Network, Cluster Network, Service Network) is undefined"},
```

Reply:
@avishayt The pattern above is regex, so . may be any character.  It is part of a test